documentation/library_guide/pstl/dpcpp_policies_usage.rst

```diff
@@ -181,6 +181,18 @@ requirements:
 * Has a ``get_buffer`` method with no arguments. The method returns the SYCL buffer passed to
   ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` functions
 
+Also ``begin``, ``end`` can optionally accept SYCL deduction tags and ``sycl::no_init``
+(See SYCL 2020: 4.7.6.3, 4.7.6.4 for details) to explicitly mention which access mode should be applied
+to the buffer accessor when submitting DPC++ kernel to a device, e.g.
+
+.. code:: cpp
+
+  auto first1 = begin(buf, sycl::read_only);
+  auto first2 = begin(buf, sycl::write_only, sycl::no_init);
+  auto first3 = begin(buf, sycl::no_init);
+
+It helps to build the dependency graph of parallel algorithms' calls more correctly.
+
```

Comment:
Maybe we can also mention somehow that parallel algorithms already set "access modes" and users can rewrite it using such way. Not sure how mention it correctly...